op-e2e/actions/proofs/sequence_window_expiry_test.go

```diff
@@ -48,6 +48,82 @@ func runSequenceWindowExpireTest(gt *testing.T, testCfg *helpers.TestCfg[any]) {
 	env.RunFaultProofProgram(t, l2SafeHead.Number.Uint64()/2, testCfg.CheckResult, testCfg.InputParams...)
 }
 
+// Runs a that proves a block in a chain where the batcher opens a channel, the sequence window expires, and then the
+// batcher attempts to close the channel afterwards.
+func runSequenceWindowExpire_ChannelCloseAfterWindowExpiry_Test(gt *testing.T, testCfg *helpers.TestCfg[any]) {
+	t := actionsHelpers.NewDefaultTesting(gt)
+	tp := helpers.NewTestParams()
+	env := helpers.NewL2FaultProofEnv(t, testCfg, tp, helpers.NewBatcherCfg())
+
+	// Mine 2 empty blocks on L2.
+	for i := 0; i < 2; i++ {
+		env.Sequencer.ActL2StartBlock(t)
+		env.Sequencer.ActL2EndBlock(t)
+	}
```

Comment:
Won't these two blocks be identical to the deposit-only blocks derived when the sequencing window expires? I guess the L1 origin may differ, but it might make sense to insert even a basic L2 transaction into them to make the difference obvious.